[2022-11-02T09:46:44.662+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: covid_pipe.download_data scheduled__2022-10-28T00:00:00+00:00 [queued]>
[2022-11-02T09:46:44.685+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: covid_pipe.download_data scheduled__2022-10-28T00:00:00+00:00 [queued]>
[2022-11-02T09:46:44.686+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2022-11-02T09:46:44.688+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 2
[2022-11-02T09:46:44.691+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2022-11-02T09:46:44.789+0000] {taskinstance.py:1383} INFO - Executing <Task(BashOperator): download_data> on 2022-10-28 00:00:00+00:00
[2022-11-02T09:46:44.822+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'covid_pipe', 'download_data', 'scheduled__2022-10-28T00:00:00+00:00', '--job-id', '745', '--raw', '--subdir', 'DAGS_FOLDER/covid_dag.py', '--cfg-path', '/tmp/tmp9ozj50u0']
[2022-11-02T09:46:44.826+0000] {standard_task_runner.py:83} INFO - Job 745: Subtask download_data
[2022-11-02T09:46:44.812+0000] {standard_task_runner.py:55} INFO - Started process 1287 to run task
[2022-11-02T09:46:45.286+0000] {task_command.py:376} INFO - Running <TaskInstance: covid_pipe.download_data scheduled__2022-10-28T00:00:00+00:00 [running]> on host 53c1f4b2cdcf
[2022-11-02T09:46:46.697+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=covid_pipe
AIRFLOW_CTX_TASK_ID=download_data
AIRFLOW_CTX_EXECUTION_DATE=2022-10-28T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-10-28T00:00:00+00:00
[2022-11-02T09:46:46.905+0000] {subprocess.py:63} INFO - Tmp dir root location: 
 /tmp
[2022-11-02T09:46:46.928+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'curl -sSL https://raw.githubusercontent.com/CSSEGISandData/COVID-19/master/csse_covid_19_data/csse_covid_19_daily_reports/01-01-2021.csv > /opt/***/01-01-2021.csv']
[2022-11-02T09:46:47.121+0000] {subprocess.py:86} INFO - Output:
[2022-11-02T09:46:49.878+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2022-11-02T09:46:49.960+0000] {taskinstance.py:1406} INFO - Marking task as SUCCESS. dag_id=covid_pipe, task_id=download_data, execution_date=20221028T000000, start_date=20221102T094644, end_date=20221102T094649
[2022-11-02T09:46:50.067+0000] {local_task_job.py:164} INFO - Task exited with return code 0
[2022-11-02T09:46:50.123+0000] {dagrun.py:683} ERROR - Failed to get task for ti <TaskInstance: covid_pipe.removedata scheduled__2022-10-28T00:00:00+00:00 [None]>. Marking it as removed.
[2022-11-02T09:46:50.151+0000] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
